src/stickerbook.js

```diff
@@ -581,6 +581,58 @@ class Stickerbook {
   }
 
   /**
+   * Places the current selected sticker programmatically on the canvas
+   * @param {Object} options An object with options for the placement
+   * @param {Number} options.x The x position at which to place the sticker
+   * @param {Number} options.y The y position at which to place the sticker
+   * @param {Number} options.xScale The x scale at which to place the sticker, defaults to 1
+   * @param {Number} options.yScale The y scale at which to place the sticker, defaults to 1
+   * @param {Number} options.rotation How much to rotate the image in degrees, defaults to 0
```

Comment:
Good question, but:
a) Screen coordinates are left-handed, meaning _rotations are clockwise_ implicitly.
b) I'm just forwarding the fields to fabric, so I'm just letting it do what it wants.

However, I can update docs to clarify that.